src/services/device.service.ts

```diff
@@ -0,0 +1,31 @@
+import { db } from '../database';
+
+export interface Device {
+  userId: number;
+  deviceToken: string;
+}
+
+interface InsertDeviceResult{
```

Comment:
Space before brace, please! ...I'm surprised our linter allows this; that seems like an oversight in our rules.

```suggestion
interface InsertDeviceResult {
```

(I just noticed that we're doing both with and without a space before the brace in `notification.service.ts`, which probably inspired this; sorry I missed that earlier. We should also fix that, but not in this PR.)